src/mobile-app/components/uploader.tsx

```diff
@@ -157,13 +157,32 @@ export const Uploader = (props: IProps) => {
       <div className={css.dialog}>
         <div className={css.header}>
           Upload Experiment
-          <div className={css.floatRight}>
-            <span onClick={props.onClose}>{uploadState === UploadState.Uploaded ? "OK" : "Cancel"}</span>
-          </div>
+
         </div>
         <div className={css.contents}>
-          <RunInfoComponent run={props.run} />
+          <div className={css.run}>
+            <div className={css.initials}><Initials text={props.run.experiment.metadata.initials}/></div>
+            <div className={css.text}>
+              <RunInfoComponent run={props.run} />
+            </div>
+          </div>
+          <div className={css.instructions}>
+            <ol>
+              <li>In the [WATERS] activity, click the Import button to display a QR code.</li>
```

Comment:
I think this is fine for now but we will need to update this once more than 1 activity uses Vortex.